### PR TITLE
Adding Node-SQL-Parser project.

### PR DESCRIPTION
**Pull request by Quinncuatro**

### CODE REVIEW

**Comment by shiftkey** [comment]
To avoid confusion over tags, CI is requesting some changes:

```
/home/runner/work/up-for-grabs.net/up-for-grabs.net/_data/projects/NodeSQLParser.yml - 
Tag verification failed!
Use 'node.js' instead of node
Use 'parsing' instead of parser
```

File: _data/projects/NodeSQLParser.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+name: Node SQL Parser
+desc: Parse simple SQL statements into an abstract syntax tree (AST) with the visited tableList and convert it back to SQL.
+site: https://github.com/taozhi8833998/node-sql-parser
+tags:
+- node
```
Comment (shiftkey) [inline _data/projects/NodeSQLParser.yml]:
```suggestion
- node.js
```

File: _data/projects/NodeSQLParser.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+- sql
+- parser
```
Comment (shiftkey) [inline _data/projects/NodeSQLParser.yml]:
```suggestion
- parsing
```

**Comment by Quinncuatro** [comment]
Thanks @shiftkey - changes are testing now. :)

**Comment by Quinncuatro** [comment]
Looks like that one went better.

**Review by shiftkey — APPROVED** [review]
@Quinncuatro thanks!

**Pull request merged**